### DISCUSSION TITLE
Copy link to selection greyed out?

### DISCUSSION THREAD

**Discussion by ac8318740** [RyotaUshio/obsidian-pdf-plus#215] [Q&A]
When I have PDF++ disabled, Obsidian lets me copy link to selection. When I have PDF++ enabled, that option is greyed out when I right click, and I have to select from the sub-menus of either "Don't specify color" or any of the colors.

Can I set up a hotkey which will copy based on the last selection (e.g., quote in callout, yellow; quote, no color, etc.)? I don't see that option in the command palette (I don't want to auto-copy, just have a hotkey that allows me to copy, then I'll turn on/off auto-paste as I see fit)

![image](https://github.com/RyotaUshio/obsidian-pdf-plus/assets/171465680/a2e56a33-3f96-4f97-a10b-e9f32085f14b)

**Comment by RyotaUshio** [comment]
Assign a hotkey to the "copy link to selection or annotation" command
